### ISSUE TITLE
Redirect to /cash after SSO registration/login

### ISSUE THREAD

**Issue by zetas** [psychobunny/nodebb-plugin-cash#7]
After users use SSO to register on my site it redirects them to the /cash route for some reason and generates an error message. 

I'm running 0.7.0-dev and I know it's not supported with this plugin but I'd like to use the openfantasy-rpg plugin which requires this. 

In the meantime I'll have to disable both of them :(

**Comment by psychobunny** [comment]
That's weird. Disable OF plugin and see if this still happens with cash mod only?

**Comment by psychobunny** [comment]
Also paste error message as well

**Comment by zetas** [comment]
It only happens with both plugins installed. Disabling OF doesn't cause the issue. Also the error displayed is: 
```
/cash Not Found
You seem to have stumbled upon a page that does not exist. Return to the home page.
```

Here's a screencap I made if it helps:
https://www.dropbox.com/s/mswh8g88qjes4gh/cash-orpg.mov?dl=0

**Comment by psychobunny** [comment]
yeah, there is no /cash route. So my wonder is why you are being redirected there.

**Comment by psychobunny** [comment]
Does everything else work btw? Your only issue is that after users use SSO they are redirected to /cash? What SSO plugin are you using?

**Comment by zetas** [comment]
Everything else works as far as I know. I'm using julianlams nodebb-plugin-sso-oauth that i modified for my custom oauth implementation.
